src/core/include/openvino/core/attribute_adapter.hpp

```diff
@@ -89,6 +89,27 @@ class DirectValueAccessor : public ValueAccessor<AT> {
     AT& m_ref;
 };
 
+#ifdef _MSC_VER
+#    define OPENVINO_API_EXTERN
+#else
+#    define OPENVINO_API_EXTERN OPENVINO_API
+#endif
+
+extern template class OPENVINO_API_EXTERN DirectValueAccessor<std::vector<std::string>>;
```

Comment:
```suggestion
extern template class OPENVINO_API DirectValueAccessor<std::vector<std::string>>;
```
Why not use OV standard export macro? They should always exported from library if they are public API

Reply:
The problem is for Linux __attribute__((visibility("default"))) required for both extern declaration and explicit instantiation.
But for Windows __declspec(dllimport) is required only for instantiation

Reply:
Maybe this macro can be in core visibility header with small description when should be used. It can be re-used instead defined it each header.

Reply:
done